packages/docs-markdown/src/test/suite/helper/metadata.test.ts

```diff
@@ -152,4 +152,31 @@ suite('Metadata Helper', () => {
 		const config = workspace.getConfiguration('markdown');
 		assert.strictEqual(config.get<boolean>('enableMetadataDateReminder'), true);
 	});
+	test('metadataDateReminder => doesnt remind on short date', async () => {
+		sinon.stub(workspace, 'getConfiguration').returns({
+			get: () => true,
+			has: () => true,
+			inspect: () => {
+				return { key: '' };
+			},
+			update: () => Promise.resolve()
+		});
+
+		const config = workspace.getConfiguration('markdown');
+		await config.update('enableMetadataDateReminder', true);
+		const stubShowInformationMessage = sinon.stub(window, 'showInformationMessage');
+		stubShowInformationMessage.onCall(0).resolves(Promise.resolve(undefined) as any);
+		const filePath = resolve(
+			__dirname,
+			'../../../../../src/test/data/repo/articles/test/metadata5.md'
+		);
+		await loadDocumentAndGetItReady(filePath);
+		const clock = sinon.useFakeTimers({
+			now: new Date(2021, 1, 2, 0, 0)
```

Comment:
is this the right date?

Reply:
Thats the date for 02/02/2021, The first parameter are as follows `new Date(year, month(zero based), day date, N/A, N/A)`